### ISSUE TITLE
Category selection in the translation function is not functioning properly.

### ISSUE THREAD

**Issue by Reticulmz** [Wynntils/Wynntils#2066]
### Minecraft and Wynntils Version

Minecraft Version: 1.19.4
Wynntils Version: 0.0.3-alpha.346
Fabric Version: 0.14.22

### Expected behavior
Translation categories should function properly
Only categories that are valid should be translated, and

### Issue being encountered
The category for the translation function only works for Info Messages, and
Player chat and NPC dialog selection is not working.
When Info Messages is enabled, player chat, NPC dialog, and info messages are all translated

### Steps to reproduce the problem

1. Open Wynntils settings
2. Open the Translation section
3. Set the Enabled toggle to Enable
4. Enable the Info Message toggle
5. All chats get translated.

**Comment by magicus** [comment]
Yeah, sorry, the translation feature is currently quite broken. We should probably disable/remove it until it gets fixed.

**Comment by ShadowCat117** [comment]
This seems to have been resolved however the translation feature is (once again) broken, needs updating for 2.1 chat

![Image](https://github.com/user-attachments/assets/f22e52a5-8672-4f97-a4e0-bc3c0e9ddc7d)
